src/main/java/com/amazon/aocagent/commands/CommonOption.java

```diff
@@ -98,4 +93,23 @@ private Stack buildStack() throws IOException, BaseException {
     return mapper.readValue(
         new String(Files.readAllBytes(Paths.get(this.stackFilePath))), Stack.class);
   }
+
+  private void setExtraContext(Map<String, String> extra, Context context) {
```

Comment:
if i want to add a new test to eks, can i achieve it by just adding a mustache config, and send this config through command?

Reply:
OK, agree